infrastructure/src/main/java/org/corfudb/infrastructure/orchestrator/Orchestrator.java

```diff
@@ -109,8 +119,13 @@ void addNode(CorfuPayloadMsg<OrchestratorRequest> msg, ChannelHandlerContext ctx
     @Nonnull
     private Workflow getWorkflow(@Nonnull OrchestratorRequest req) {
         Request payload = req.getRequest();
-        if (payload.getType().equals(Types.OrchestratorRequestType.ADD_NODE)) {
-            return new AddNodeWorkflow(payload);
+        switch (payload.getType()) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Complete cases by adding the missing enum constants or add a default case to this switch. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)